src/az390.java

```diff
@@ -2102,10 +2103,32 @@ else if (tz390.op_trace_type[index]==202)                          //      #613
                 case 23:
                     my_format="RIE";                                                   // #500
                     my_operands="R1,R3,RI2";                                           // #500 #561
+                    if (tz390.op_trace_type[index]==230)                               // #614
```

Comment:
Magic number 230. Many more such follow. Leave for now but make note to create constants???